### DISCUSSION TITLE
اخبار : مشکل وی توری و وایرگارد روی همراه اول

### DISCUSSION THREAD

**Discussion by upgo7502** [Gozargah/Marzban#859] [General]
با سلام خدمت همه دوستان

جدیدا روی همراه اول برای سرویس های وی تو ری و وایگارد محدودیتی اعمال شده که بعضی از کاربران در حین استفاده دچار مشکل قطعی میشن و باید یک بار گوشی رو air mode میکنن و بردارن تا آی پی جدید بهشون بده تا بتونن وصل بشن . این موضوع روی ایرانسل هم جدید توی 24 ساعت اخیر پیاده سازه شده و کم کم داره کستش پیدا میکنه . کسی برای این موضوع راه حلی داره ؟

نکته : کاربرایی که پهنای باند بالای 10 مگ مصرف میکنن و با سرعت بالایی کار میکنن بصورت رندوم اینطوری میشن .
توی 5 روز اخیر خیلی از این گزارشها داشتم و خودمم تست کردم دقیقا همینه . حتی روی سرور های خودم و سرور های دیگران هم تست کردم مخصوصا روی شبکه همراه اول این موضوع دیده میشه . بین 100 تا کاربر تقریبا 4 الی 5 نفر ممکنه در روز به مشکل بخورن .

پیروز باشید

**Comment by amirdana** [comment]
منم چند ماهی میشه این مشکل رو دارم، اخیرا خیلی بیشتر هم شده، بیشتر فکر‌ میکنم مربوط به QOS باشه تا فیلترینگ ولی بازم مطمئن نیستم

**Comment by upgo7502** [comment]
مخصوصا روی UDP ها بیشتر شده
